main.c

```diff
@@ -101,6 +101,7 @@ void main(void){
     vector_init(&tests_to_run);
     load_conf_file("D:\\config.txt");
     open_output_file("D:\\kernel_tests.log");
+    srand(NULL);
```

Comment:
It should also be considered to reseed before every single test, so that if new tests are added (after seed, before old tests), it won't affect the old test results (which would now get a new random stream if then new test consumed random bits). This should probably be done in this PR.

In fact, one could prevent this issue by desiging an API where you have to create a random object (`Random* create_randomness(unsigned int seed_offset);`) and query values through that (`int get_random(Random* randomness)`). Thereby forcing to user to reseed manually.
`rand()` by itself would be forbidden in the codebase then. However, I feel this is beyond what this PR aims to do.